arch/xtensa/Kconfig

```diff
@@ -173,3 +173,7 @@ endif # XTENSA_MMU
 endif # CPU_HAS_MMU
```

Comment:
For some reason for imx we used 3072 and it works fine :). Let's keep it that way.

Cc: @iuliana-prodan we should at some point look at this and see if we can shrink it to 2048 as everyone does.

Reply:
we'll need to check with thread analyzer.